addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,14 +2,15 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl" owl="1">
-        <div t-attf-class="o_notification {{className}}"
+        <div t-attf-class="o_notification toast show border-{{props.type}} overflow-hidden mb-2"
         role="alert" aria-live="assertive" aria-atomic="true">
-            <div class="o_notification_body">
-                <button type="button" class="close o_notification_close"
-                aria-label="Close" t-on-click="props.close">
-                    <span class="d-inline" aria-hidden="true">×</span>
+            <div t-attf-class="toast-header align-items-start justify-content-end pb-0 ">
```

Comment:
Why `align-items-start`?

Reply:
In case the title has two lines (if that happens..), the close button (being in the header with the title) will always remain at the top

Reply:
Ok, I got it :+1: